tcms/testruns/tests/test_admin.py

```diff
@@ -63,6 +64,10 @@ def setUpTestData(cls):
         super().setUpTestData()
         initiate_user_with_default_setups(cls.tester)
 
+    positive = TestExecutionStatus.objects.all().filter(weight__gt=0)
+    neutral = TestExecutionStatus.objects.all().filter(weight=0)
+    negative = TestExecutionStatus.objects.all().filter(weight__lt=0)
```

Comment:
I was going to say that this was illegal syntax but it doesn't crash because Python considers these variables as class attributes b/c they are outside of any method but inside the class body. Probably not what you wanted.